[Docs] Update threat score references (#22210)
cloudflare/cloudflare-docs: src/content/docs/support/third-party-software/content-management-system-cms/cloudflare-and-joomla-recommended-first-steps.mdx

---

## Skip security features for specific IP addresses

You can use WAF custom rules to [skip certain security features](/waf/custom-rules/skip/) for IP addresses you want traffic from or expect traffic from. Some common services you probably want to allow include:

- APIs you are getting data from
- Monitoring services you use to monitor your site's uptime
- Security services
- IP addresses you frequently log in from

Why do this?

> If you have enabled and configured [Cloudflare's Web Application Firewall](/waf/), you may get challenged while working on your backend and/or services you want to access your site may get challenged. Taking the steps to skip certain features for requests coming from known IP addresses will help prevent issues on your site.

Reviewer: ```suggestion
If you have enabled and configured [Cloudflare's Web Application Firewall](/waf/), legitimate services and/or you may get challenged while accessing your backend. To prevent issues on your site, consider skipping certain features for requests coming from known IP addresses.
```

Reviewer: WAF features?

Author: Not just from WAF, but other security features.

:::note
Cloudflare allows known search engine and social media crawlers. If you decide to block specific countries, you must use care because you may end up inadvertently blocking their crawlers (blocking the USA, for example, could mean that their crawler gets challenged).
:::

---

## Ensure requests from Cloudflare's IP ranges aren't blocked or limited

> If you are using services like `.htaccess`, firewalls or server mods to manage access to your site from visitors, it is important to make sure requests from Cloudflare’s IP ranges are not being blocked or limited in any way. The number one cause of site offline issues in Cloudflare's support channel is something blocking or restricting requests from Cloudflare IPs, so please take the time to make sure that all of Cloudflare's IPs are allowed on your server.

Reviewer: ```suggestion
If you are using services like `.htaccess`, firewalls or server mods to manage access to your site from visitors, you must ensure that requests from Cloudflare’s IP ranges are not being blocked or limited in any way. The number one cause of "site offline" issues in Cloudflare's support channel is something blocking or restricting requests from Cloudflare IPs. To prevent this, ensure that all of Cloudflare's IPs are allowed on your server.
```
maybe link to that Cloudflare IP list in the sentence if you have it
